File: Modules/Common/Resources/SIE.Web.Resources/Employees/Commands/EmployeeResourceImportCommand.js
SIE.defineCommand('SIE.Web.Resources.Employees.Commands.EmployeeResourceImportCommand', {
    extend: 'SIE.Web.Common.Import.Commands.ImportExcelCommand',
    meta: { text: "导入", hierarchy: "导入".t(), group: "business", iconCls: "icon-Upload icon-green" }
});

File: Modules/Common/Resources/SIE.Web.Resources/Employees/Commands/EmployeeResourceImportCommand.cs
using SIE.Common.Import;
using SIE.Web.Common.Import.Commands;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SIE.Web.Resources.Employees.Commands
{
    public class EmployeeResourceImportCommand: ImportExcelCommand
    {
        /// <summary>
        /// 保存
        /// </summary>
        /// <param name="batch"></param>
        protected override void OnSave(IList<RowData> batch)
        {
           
            base.OnSave(batch);
        }
    }
}
